src/mpid/ch4/subconfigure.m4

```diff
@@ -236,21 +236,13 @@ if test "$ch4_nets_array_sz" = "1" && (test "$enable_ch4_netmod_inline" = "yes"
    PAC_APPEND_FLAG([-DNETMOD_INLINE=__netmod_inline_${ch4_netmods}__], [CPPFLAGS])
 fi
 
-AC_ARG_ENABLE(ch4-direct,
-    [--enable-ch4-direct
-       Defines the direct communication routine used in CH4 device
```

Comment:
It might be better to still show the option but list it as deprecated. So the user knows what the replacement is. We did it for `ch4-netmod-direct`.

Reply:
No, if we retain that option, my preference would be to throw an error.  That way users will stop using it.

Reply:
I've changed this to throw an error.  Our jenkins tests will fail because of this.  We'll need to update them.